### ISSUE TITLE
timer and pot not showing

### ISSUE THREAD

**Issue by chiiledgamer1985** [ztizzlegaming/CSGOWinBig#77]
after playing about with it for the last 2hrs i got the site to work this far 

http://prntscr.com/95b8ux

but trying to get the pot to show with the timer to 
all the chat works fine and saves the steam trade urls so as far as i can work out its not database side of things 

any idea's ?

**Comment by Stuffa1991** [comment]
Do you have the bot running?

**Comment by chiiledgamer1985** [comment]
yes bot is running tho cant trade with it for 7 days due to only signed in with it for the 1st time on my  dedicated server

**Comment by TheAnthonyNL** [comment]
Ehm for me everything is loading tho... pot starts when someone goes in and timer when another one joins the pot.

**Comment by chiiledgamer1985** [comment]
oh so its when someone bets the pot comes up i didnt no that XD sorry guys
i fort it was like some where it loads up with out anyone bettings
